assemblers/io.weldr.qcow2

```diff
@@ -74,7 +74,7 @@ def main(tree, input_dir, options):
                 with mount_api(mountpoint):
                     subprocess.run(["chroot", mountpoint, "grub2-install", "--no-floppy", "--target", "i386-pc", loop], check=True)
 
-        subprocess.run(["qemu-img", "convert", "-O" "qcow2", "-c", image, filename], check=True)
+        subprocess.run(["qemu-img", "convert", "-O" "qcow2", "-c", image, f"/run/osbuild/output/{filename}"], check=True)
```

Comment:
The patch is correct, but please use the "output_dir" option like the tree assembler does, so we don't rely on the location of the output tree being fixed.

Reply:
Actually the output directory is binded to this one in the nspawn container. So this way we are using it:
https://github.com/larskarlitski/osbuild/blob/master/osbuild.py#L149

Reply:
Yeah, it works the same at the moment. However, we put that path in the output_dir variable in the options array, so that in the future we could change the bindmount location in osbuild.py without having to update any of the stages.